makefile

```diff
@@ -1533,7 +1533,7 @@ generate: \
 
 $(GENDIR)/includes/SDL2:
 	-$(call MKDIR,$@)
-	-$(call COPY,3rdparty/SDL2/include/,$(GENDIR)/includes/SDL2)
+	-$(call COPY,3rdparty/SDL2/include/.,$(GENDIR)/includes/SDL2)
```

Comment:
Does this change work on *NIX (e.g. Linux) as well as Windows?  I'm kinda cagey about this kind of thing, the copy rule is not very safe due to having to hack around potentially not having `cp` on Windows.  Does it work if you remove the slash rather than adding the dot?